src/helpers/ResolveRundownIntoPlaylist.ts

```diff
@@ -73,48 +73,47 @@ function isKlarOnAir(segment: UnrankedSegment): boolean {
 	return !!segment.name?.match(klarOnAirPattern)
 }
 
-function extractAndSetGraphicProfile(segment: UnrankedSegment, rundown: ResolvedPlaylistRundown): void {
-	const graphicProfiles = getOrderedGraphicProfiles(segment)
-	if (graphicProfiles.length > 0) {
-		const graphicProfile = graphicProfiles[0]
-		setGraphicsProfile(rundown, graphicProfile)
+function extractAndSetShowstyleVariant(segment: UnrankedSegment, rundown: ResolvedPlaylistRundown): void {
```

Comment:
Minor thing. 
Having an "and" in the method is a clear indicator that the method does more than one thing and therefore violates SRP ;)